WordPress/src/main/java/org/xmlrpc/android/XMLRPCUtils.java

```diff
@@ -289,6 +297,9 @@ private static String verifyXmlrpcUrl(final String siteUrl, final String httpUse
                     // Endpoint found and works fine.
                     return url;
                 }
+            } catch (XMLRPCUtilsException e) {
```

Comment:
Unfortunately this breaks support for self hosted sites with self signed certificate.